litmus-portal/frontend/src/components/WelcomeModal/styles.ts

```diff
@@ -1,95 +1,76 @@
 import { makeStyles, Theme } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
-  root: {
-    '& > *': {
-      margin: theme.spacing(1),
-      // width: 350,
-      outline: 'none',
-      marginTop: 30,
-    },
-  },
+  /* CSS for Index */
   rootContainer: {
-    marginLeft: '21rem',
-    marginRight: '21rem',
-    marginTop: '7.8rem',
-    marginBottom: '7.8rem',
-    paddingBottom: '6rem',
+    margin: '3rem auto',
```

Comment:
This can be changed for the top and bottom margins to use the spacing from theme.

Reply:
In this case I'm in favour of using rem...as 3rem is a good number but theme.spacing(<some fraction below zero>) is not

Reply:
Yeah it may be changed not necessarily.